<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>700</width>
     <height>500</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_5">
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(231, 231, 231);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QLabel" name="label">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>10</y>
               <width>601</width>
               <height>51</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);</string>
             </property>
             <property name="text">
              <string>Cadastro de Motorista</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
            <widget class="QLabel" name="label_2">
             <property name="geometry">
              <rect>
               <x>80</x>
               <y>80</y>
               <width>41</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>Nome</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_3">
             <property name="geometry">
              <rect>
               <x>90</x>
               <y>130</y>
               <width>31</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>CPF</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_4">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>180</y>
               <width>81</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>Placa Veiculo</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_5">
             <property name="geometry">
              <rect>
               <x>290</x>
               <y>180</y>
               <width>31</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>CNH</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_6">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>230</y>
               <width>111</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>Vencimento CNH</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_7">
             <property name="geometry">
              <rect>
               <x>330</x>
               <y>230</y>
               <width>47</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>Categoria</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_8">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>280</y>
               <width>101</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>Validade Seguro</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit">
             <property name="geometry">
              <rect>
               <x>120</x>
               <y>80</y>
               <width>400</width>
               <height>35</height>
              </rect>
             </property>
             <property name="maxLength">
              <number>50</number>
             </property>
             <property name="placeholderText">
              <string>  Nome e sobrenome</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit_2">
             <property name="geometry">
              <rect>
               <x>120</x>
               <y>130</y>
               <width>281</width>
               <height>35</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);</string>
             </property>
             <property name="inputMethodHints">
              <set>Qt::ImhNone</set>
             </property>
             <property name="inputMask">
              <string/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="maxLength">
              <number>11</number>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Normal</enum>
             </property>
             <property name="placeholderText">
              <string>  Digitar o CPF sem . e - Ex =  '12312312312'</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>false</bool>
             </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit_3">
             <property name="geometry">
              <rect>
               <x>120</x>
               <y>180</y>
               <width>111</width>
               <height>35</height>
              </rect>
             </property>
             <property name="maxLength">
              <number>7</number>
             </property>
             <property name="placeholderText">
              <string>  Ex: ttt8585</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit_4">
             <property name="geometry">
              <rect>
               <x>329</x>
               <y>180</y>
               <width>191</width>
               <height>35</height>
              </rect>
             </property>
             <property name="maxLength">
              <number>11</number>
             </property>
             <property name="placeholderText">
              <string>  Numero CNH -11 Digitos</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit_5">
             <property name="geometry">
              <rect>
               <x>120</x>
               <y>230</y>
               <width>200</width>
               <height>35</height>
              </rect>
             </property>
             <property name="maxLength">
              <number>8</number>
             </property>
             <property name="placeholderText">
              <string>  Digitar data sem - e / - Ex: 12122020</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit_6">
             <property name="geometry">
              <rect>
               <x>120</x>
               <y>280</y>
               <width>411</width>
               <height>35</height>
              </rect>
             </property>
             <property name="inputMask">
              <string/>
             </property>
             <property name="maxLength">
              <number>32767</number>
             </property>
             <property name="placeholderText">
              <string>    Digitar data sem - e / - Ex: 12122020</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit_7">
             <property name="geometry">
              <rect>
               <x>390</x>
               <y>230</y>
               <width>130</width>
               <height>35</height>
              </rect>
             </property>
             <property name="maxLength">
              <number>2</number>
             </property>
             <property name="placeholderText">
              <string> Categoria da CNH</string>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton">
             <property name="geometry">
              <rect>
               <x>254</x>
               <y>380</y>
               <width>115</width>
               <height>51</height>
              </rect>
             </property>
             <property name="text">
              <string>Cadastrar</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
